docs/NOW-spec.md

```diff
@@ -910,7 +588,7 @@ The NOW_SESSION_MSGBOX_REQ_MSG is used to show a message box in the user session
 
 | Flag                                | Meaning                                 |
 |-------------------------------------|-----------------------------------------|
-| NOW_MSGBOX_FLAG_TITLE<br>0x00000001 | The title field contains a non-default value |
+| NOW_MSGBOX_FLAG_TITLE<br>0x00000001 | Set if `title` field is present |
```

Comment:
Guarded some optional fields via flags in several messages, so they could be skipped from serialization/deserialization